lt-ud-train.conllu

```diff
@@ -2819,7 +2819,7 @@
 # sent_id = 101
 # text = Vytautas Radžvilas kalba apie Lietuvos išvalstybinimą, nutautinimą, valstybinės ir tautinės sąmonės dekonstrukciją Europos Sąjungoje.
 1	Vytautas	Vytautas	PROPN	NNP	Case=Nom|Gender=Masc|Number=Sing	2	nsubj	_	En=Vytautas
-2	Radžvilas	Radžvilas	PROPN	NNP	Case=Nom|Gender=Masc|Number=Sing	3	flat	_	En=Radžvilas
+2	Radžvilas	Radžvilas	PROPN	NNP	Case=Nom|Gender=Masc|Number=Sing	3	appos	_	En=Radžvilas
```

Comment:
Both flat and appos should be head initial, so obviously there was an error in the original data and there is another error in this new data.
For this reason it is listed in [bugs.html](http://ufallab.ms.mff.cuni.cz/~popel/lt/bugs.html) as a ToDo for the next release.

I don't understand this sentence, where the main predicate "kalba" is annotated as NOUN (shouldn't it be a VERB as the features suggest?).
If  "Vytautas Radžvilas kalba apie" means "Vytautas Radžvilas speaks about", you are right Radžvilas should have deprel=flat (or flat:name), but it should not depend on kalba, but on Vytautas. Once this is fixed in the original data, my script won't change it to appos anymore.